classification/fasttext/code/notebooks/fasttext_nb.ipynb

```diff
@@ -633,23 +634,24 @@
     "        project=f\"{WORKSPACE_NAME}/{PROJECT_NAME}\",\n",
     "        name=\"Fasttext text classification\",\n",
     "        description=\"Optuna tuned fasttext text classification\",\n",
-    "        tags=[\"fasttext\", \"optuna\", \"trial-level\", \"notebook\", \"sagemaker\"],\n",
+    "        tags=[\"fasttext\", \"optuna\", \"trial-level\", \"notebook\"],\n",
     "    )\n",
     "\n",
     "    # (neptune) log sweep id to trial-level Run\n",
     "    run_trial_level[\"sweep_id\"] = sweep_id\n",
     "\n",
-    "    # (neptune) log parameters of a trial-level Run\n",
+    "    # train model on selected hyperparams\n",
```

Comment:
`(neptune)` -> we need this exact to demo purposes

Reply:
`(neptune) log parameters of a trial-level Run`
I've just moved this comment below the training line